runtime/near-vm-runner/src/logic/errors.rs

```diff
@@ -210,6 +210,8 @@ pub enum HostError {
     YieldPayloadLength { length: u64, limit: u64 },
     /// Yield resumption data id is malformed.
     DataIdMalformed,
+    /// Size of the recorded trie storage proof has exceeded the allowed limit.
+    RecordedStorageExceeded { size: usize, limit: usize },
```

Comment:
nit: best to use bytesize for type safety and human friendliness

Reply:
Yeah with a single value it looks much more reasonable:
```
Error: 
   0: Error: An error occurred during a `FunctionCall` Action, parameter is debug message.
      ExecutionError("Size of the recorded trie storage proof has exceeded the allowed limit (4.0 MB)")


```

Reply:
I hope it's okay to add a `bytesize` dependency to `near-vm-runner` :eyes: